runtime/layers/fpm-dev/php-fpm.conf

```diff
@@ -1,5 +1,5 @@
 ; Logging anywhere on disk doesn't make sense on lambda since instances are ephemeral
-error_log = /dev/stderr
+error_log = /dev/null
```

Comment:
I set the same value as in the production version. Logs are collected via `--force-stderr`, just like on Lambda (one less difference).